pom.xml

```diff
@@ -156,7 +162,6 @@
 		<dependency>
 			<groupId>org.mockito</groupId>
 			<artifactId>mockito-core</artifactId>
-			<!-- see https://github.com/mockito/mockito/issues/1606 -->
```

Comment:
Maybe we want to add a similar comment to the new ByteBuddy dependency to explain why we need this workaround?